docs/quickstart.md

```diff
@@ -132,14 +126,14 @@ If the installation steps fail, it might be a dependency issue. Make sure you ha
 
 ## Setup author name
 
-AEAs are composed from components. The components can be developed by anyone and are available on the <a href="https://aea-registry.fetch.ai" target="_blank">AEA registry</a>. To use the registry we need to register an author name.
+AEAs are composed from components. AEAs and AEA components can be developed by anyone and pushed to the <a href="https://aea-registry.fetch.ai" target="_blank">AEA registry</a> for others to use. To use the registry, we need to register an author name.
 
-You can setup your author name using the `init` command:
+You can set up your author name using the `init` command:
 ``` bash
 aea init
 ```
 
-This is your unique author name in the Fetch.ai ecosystem.
+This is your unique author (or developer) name in the AEA ecosystem.
 
 You should see a similar output (with your input instead of the sample username and email):
 ``` bash
```

Comment:
In “Do you have a Registry account? [y/N]” line below, why is the y and N different cases?